sdk/include/reactos/idl/winspool.idl

```diff
@@ -74,6 +74,8 @@ typedef struct _WINSPOOL_DOC_INFO_1 {
 }
 WINSPOOL_DOC_INFO_1;
 
+/* FIXME: Why are WINSPOOL_DOC_INFO_2 and WINSPOOL_DOC_INFO_3 missing? */
```

Comment:
These don't need to start a converstaion. Just the struct name, maybe with the word "missing" is enough.
Unless you're not sure that they're missing, in which case, be sure or don't bother adding comments.

Reply:
> Just the struct name

Done. Added one more case.